src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -419,6 +421,13 @@ internal void WorkerInitResponse(GrpcEvent initEvent)
                 _isWorkerApplicationInsightsLoggingEnabled = true;
             }
 
+            // If http proxying is enabled, we need to get the proxying endpoint of this worker
+            var httpUri = _workerCapabilities.GetCapabilityState(RpcWorkerConstants.HttpUri);
```

Comment:
This capability is currently called "HttpUri" - it's the uri string sent from the worker through the capabilities. We will forward http requests to this endpoint. Other considered names: HttpProxyUri, HttpTriggerUri, etc.